detection_rules/kbwrap.py

```diff
@@ -150,6 +150,16 @@ def parse_list_id(s: str):
         click.echo(f' - {rule_str}')
     if response['errors']:
         handle_response_errors(response)
+    else:
+        def process_dicts(dicts, message, key):
```

Comment:
Can you make the function variables a bit more descriptive? Also prefix this method with `_` (should probably do this for parse_list_id as well.

Reply:
Updated to be more descriptive and added "_" prefix :+1: